spec/requests/admin/druid_search_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Search for druid' do
+  describe 'GET /admin/druid_search/new' do
+    context 'when the user is not authorized' do
+      before do
+        sign_in(create(:user))
+      end
+
+      it 'redirects to root' do
+        get '/admin/druid_search/new'
+
+        expect(response).to redirect_to(root_path)
+      end
+    end
+
+    context 'when the user is an admin' do
+      before do
+        sign_in(create(:user), groups: ['dlss:hydrus-app-administrators'])
+      end
+
+      it 'show search form' do
+        get '/admin/druid_search/new'
+
+        expect(response).to have_http_status(:ok)
+      end
+    end
+  end
+
+  describe 'GET /admin/druid_search/search' do
+    context 'when the user is not authorized' do
+      before do
+        sign_in(create(:user))
+      end
+
+      it 'redirects to root' do
+        get '/admin/druid_search/search?admin_druid_search[druid]=foo&commit=Submit'
+
+        expect(response).to redirect_to(root_path)
+      end
+    end
+
+    context 'when the user is an admin' do
+      before do
+        sign_in(create(:user), groups: ['dlss:hydrus-app-administrators'])
+      end
+
+      context 'when no results' do
+        it 'show search results' do
+          get '/admin/druid_search/search?admin_druid_search[druid]=foo&commit=Submit'
+
+          expect(response).to have_http_status(:unprocessable_entity)
+          expect(response.body).to include('not found')
+        end
+      end
+    end
```

Comment:
Since there's no setup for this `context` block, and there's no test in the surrounding block here for when there *are* results, I'd be tempted to dump the inner `context` block and amend the surrounding `context` with more text. That sounded confusing when I typed it out, so _in lieu_ of all that text:

```suggestion
    context 'when the user is an admin and no results' do
      before do
        sign_in(create(:user), groups: ['dlss:hydrus-app-administrators'])
      end

      it 'show search results' do
        get '/admin/druid_search/search?admin_druid_search[druid]=foo&commit=Submit'

        expect(response).to have_http_status(:unprocessable_entity)
        expect(response.body).to include('not found')
      end
    end
```